finalfrontier-utils/src/util.rs

```diff
@@ -25,78 +29,42 @@ static MINCOUNT: &str = "mincount";
 static MINN: &str = "minn";
 static MAXN: &str = "maxn";
 static MODEL: &str = "model";
+static NO_TYPED_DEPS: &str = "untyped";
```

Comment:
This read a bit weirdly, since some people also use NO for N (number). Maybe just `UNTYPED_DEPS`?

Reply:
Yeah, definitely not in line with `"untyped"` that's used everywhere else.